src/selectors/getAllProjectKeys.js

```diff
@@ -1,10 +1,26 @@
 import {createSelector} from 'reselect';
+import isNull from 'lodash-es/isNull';
 
 export default createSelector(
   state => state.get('projects'),
   projects => projects.
-    filter(project => project.updatedAt).
-    sortBy(project => -project.updatedAt).
+    sort((...args) => {
+      const [firstProject, secondProject] = args;
+      const firstProjectUpdatedAt = firstProject.updatedAt;
+      const secondProjectUpdatedAt = secondProject.updatedAt;
+      const isFirstProjectNull = isNull(firstProjectUpdatedAt);
+      const isSecondProjectNull = isNull(secondProjectUpdatedAt);
+      if (isFirstProjectNull && isSecondProjectNull) {
+        return 0;
```

Comment:
Forget if I mentioned this already but you could nest this inside the below `if` statement, so:

```js
if (isFirstProjectNull) {
  if (isSecondProjectNull) {
    return 0;
  }
  return -1;
}
if (isSecondProjectNull) {
  return 1;
}
// etc.
```

Reply:
Yeah I was thinking of something like this, but then I thought the code would be unclear this way.